blaise_survey_manager/users/views.py

```diff
@@ -136,15 +136,14 @@ def delete_user():
             db.session.delete(user)
             db.session.commit()
             send_to_rabbit_exchange('user_details', json.dumps(payload))
-            flash("User: " + user.firstname + " " + user.surname + " (" + user.username + ")  has been deleted", "success")
-
+            flash(f"User: {user.firstname or 'unknown'} {user.surname or 'unknown'} ({user.username or 'unknown'}) has been deleted", "success")
         return redirect(url_for(".list_users", _external=True, _scheme=(os.getenv('URL_SCHEME', ''))))
 
     if request.method == 'POST' and confirm is None:
         confirm = 'not-selected'
 
     previous_page = url_for(".list_users")
-    item_name = user.firstname + " " + user.surname + " (" + user.username + ")"
+    item_name = f"{user.username}"
```

Comment:
```suggestion
    item_name = f"{user.firstname or 'unknown'} {user.surname or 'unknown'} ({user.username})"
```
Can we have the first and last name in the Delete Page, always good to make it clear.

Reply:
oooh, shoot! I missed that one! My bad, I'm on it!